### PR TITLE
added open app button for mobile

### PR DESCRIPTION
**Pull request by okeinom**

### CODE REVIEW

**Comment by okeinom** [comment]
https://audioverse7.atlassian.net/jira/software/projects/AV/boards/3?assignee=712020%3Afbf78bba-2174-45a3-bda3-9bd364fbc1ee&selectedIssue=AV-804

File: src/components/molecules/openAppButton.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { FormattedMessage } from 'react-intl';
+
+import Button from './button';
+import styles from './openAppButton.module.scss';
+
+const OpenAppButton: React.FC = () => {
+	const [isMobile, setIsMobile] = useState<boolean>(false);
+	const [os, setOs] = useState<string | null>(null);
+
+	useEffect(() => {
```
Comment (alangumer) [inline src/components/molecules/openAppButton.tsx:11]: Remove this and move isIOS and isAndroid out, and do the validations based on those values.

**Review by alangumer — CHANGES_REQUESTED** [review]

File: src/components/molecules/openAppButton.tsx (hunk 1 above)
Comment (okeinom) [reply src/components/molecules/openAppButton.tsx:11]: I did a small update, however, we need the useEffect to ensure the userAgent runs in the client side and that navigator can be found.

Comment (alangumer) [reply src/components/molecules/openAppButton.tsx:12]: I see, ok.

**Review by alangumer — APPROVED** [review]

**Pull request merged**